pkg/scheduler/conf/scheduler_conf.go

```diff
@@ -49,4 +49,6 @@ type PluginOption struct {
 	PredicateDisabled bool `yaml:"disablePredicate"`
 	// NodeOrderDisabled defines whether NodeOrderFn is disabled
 	NodeOrderDisabled bool `yaml:"disableNodeOrder"`
+	// Arguments defines the different arguments that can be given to different plugins
+	Arguments map[string]string `yaml:"arguments"`
```

Comment:
We need to pass the arguments to plugins: update plugin's `New` to `New(args map[string]string)`.

Reply:
Have made those changes